tensorflow/core/api_def/base_api/api_def_ZipDataset.pbtxt

```diff
@@ -1,5 +1,29 @@
 op {
   graph_op_name: "ZipDataset"
   visibility: HIDDEN
+  in_arg {
+    name: "input_datasets"
+    description: <<END
+List of `N` variant Tensors representing datasets to be zipped together.
+END
+  }
+  attr {
+    name: "output_types"
+  }
+  attr {
+    name: "output_shapes"
+  }
+  attr {
+    name: "N"
+    description: "The length of `input_datasets`"
+  }
   summary: "Creates a dataset that zips together `input_datasets`."
+  description: <<END
+The resulting dataset will produce a stream of elements, whose elements are 
```

Comment:
replace `stream of elements` with a `dataset` ... let's not introduce redundant synonyms

Reply:
Reworded this sentence in commit 771666c